Cargo.toml

```diff
@@ -20,11 +20,16 @@ faster-hex = "0.3"
 
 [dev-dependencies]
 byteorder = "1.3.1"
-ckb-core = { git = "https://github.com/nervosnetwork/ckb.git", rev = "e718d4c046669b6e49d4673bdc2a16e08e7609f9" }
-ckb-script = { git = "https://github.com/nervosnetwork/ckb.git", rev = "e718d4c046669b6e49d4673bdc2a16e08e7609f9" }
-ckb-crypto = { git = "https://github.com/nervosnetwork/ckb.git", rev = "e718d4c046669b6e49d4673bdc2a16e08e7609f9" }
-ckb-dao-utils = { git = "https://github.com/nervosnetwork/ckb.git", rev = "e718d4c046669b6e49d4673bdc2a16e08e7609f9" }
-ckb-hash = { git = "https://github.com/nervosnetwork/ckb.git", rev = "e718d4c046669b6e49d4673bdc2a16e08e7609f9" }
+ckb-core = { path = "../ckb/core" }
```

Comment:
🤔

Reply:
Yep that was a mistake from local debugging, will fix it in a bit